receiver/postgresqlreceiver/testdata/integration/expected_all_db.json

```diff
@@ -1884,7 +1884,7 @@
                                         ],
                                         "startTimeUnixNano": "1662565933924183000",
                                         "timeUnixNano": "1662565943934293000",
-                                        "asInt": "16"
+                                        "asDouble": "16"
```

Comment:
Since this is now a double should more test cases be added that test a double?

Reply:
Yes, good call.